apps/ledger-live-mobile/src/newArch/features/Accounts/screens/AddAccount/types.ts

```diff
@@ -2,12 +2,16 @@ import { CryptoCurrency, TokenCurrency } from "@ledgerhq/types-cryptoassets";
 import { ScreenName } from "~/const";
 import { Device } from "@ledgerhq/types-devices";
 
+type CommonParams = {
+  context?: "addAccounts" | "receiveFunds";
```

Comment:
Enum?

Reply:
string literal typing are more useful in this case for autocompletion. 
If you use an enum, typscript won't be able to infer the value of the context. 
Just used the enum in live-common as it's more adapted